awc/src/request.rs

```diff
@@ -586,16 +586,16 @@ mod tests {
     use super::*;
     use crate::Client;
 
-    #[test]
-    fn test_debug() {
+    #[actix_rt::test]
```

Comment:
I can't see clearly these tests include using multiple clients in parallel.  I am assuming this is where the `ConnectorPool` issues will show up, but I could be wrong.

Reply:
Because the client now creates a future when it is launched, instead of one per request, these tests now require actix_rt. But the tests themselves don't really use the connectorpool at all.

The benches on the other hand hammer it hard.